grub.cfg

```diff
@@ -0,0 +1,10 @@
+set timeout=10
+set default=0
+
+set menu_color_normal=white/black
+set menu_color_highlight=white/green
+
+menuentry "Ironic Inspector" {
+  linuxefi ironic-python-agent.kernel ipa-inspection-callback-url=http://IRONIC_IP:5050/v1/continue ipa-inspection-collectors=default,extra-hardware,logs systemd.journald.forward_to_console=yes BOOTIF=${net_default_mac} ipa-debug=1 ipa-inspection-dhcp-all-interfaces=1 ipa-collect-lldp=1 initrd=ironic-python-agent.initramfs
+  initrdefi ironic-python-agent.initramfs
+}
```

Comment:
Using grub is moderately concerning given there is no real fallback capability and separate tftp targets will have to be used... and it REALLY wants to use tftp by default. :\

to run grub for ironic itself, you'll need to change the default boot interface to pxe.

Reply:
We could stick with a custom iPXE rom for virt, which we try and figure out why chain loading on OVMF is stalling

Reply:
I'm happy to drop it, I just didn't make any progress on figuring out the stall, and it was nice that grub2 just worked.